# std_msgs/Header
# Pose -> cv::Vec3d for position and orientation
# std::vector<int> ids
# std::vector<float> sizes

Header header
geometry_msgs/Pose[] poses
int32[] ids
float64[] sizes